### PR TITLE
Catalog: Bucket permissions: GraphQL

### PR DESCRIPTION
**Pull request by nl0**
# Description

# TODO

- [x] schema
- [x] model definition
- [x] wire up admin ui mutations
- [x] write updaters / optimistic results for mutations
- [x] normalize form input values (trim, etc)
- [x] fetch roles for users table via graphql
- [x] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

File: catalog/app/containers/Admin/RFForm.tsx
Comment (fiskus) [inline catalog/app/containers/Admin/RFForm.tsx:8]: I see, that you are in a hurry :)
So, I'm approving now and review later, even after merging

**Review by fiskus — APPROVED** [review]

File: catalog/app/containers/Admin/RFForm.tsx
Comment (nl0) [reply catalog/app/containers/Admin/RFForm.tsx:8]: the problem with this is more fundamental -- we need a way to define form typings properly, so every field is strongly typed wrt field value, validator, initial value, errors and such (smth like this probably: https://github.com/ForbesLindesay/define-form/tree/master/packages/react-define-form), otherwise it doesnt make much sense

**Pull request merged**
